dev/com.ibm.ws.config/bnd.bnd

```diff
@@ -95,4 +95,6 @@ instrument.classesExcludes: com/ibm/ws/config/internal/resources/*.class
 	com.ibm.ws.kernel.service;version=latest, \
 	com.ibm.ws.logging;version=latest, \
 	com.ibm.ws.kernel.boot.logging;version=latest, \
-	com.ibm.ws.kernel.security.thread;version=latest
+	com.ibm.ws.kernel.security.thread;version=latest,\
+	com.ibm.ws.kernel.boot.core;version=latest,\
```

Comment:
Did you need both core and common here?

Reply:
> Did you need both core and common here?

This is the testpath.  The unit tests will fail without core on its classpath during execution.